docs/docs/installation_guide/installation/cloud9install.md

```diff
@@ -100,8 +100,8 @@ To install long-term support version, enter:
 
 <a name="script"></a>
 
-`scripts/environment-deploy.sh <stage>`
-Example: scripts/environment-deploy.sh dev
+`scripts/environment-deploy.sh <stage> cn-north-1|cn-northwest-1`
```

Comment:
The region parameter is not mandatory for global region, but it is mandatory for china regions. because we need this parameter to check whether the lambda edge need to be skipped.